test/dfusion_multi_safes.js

```diff
@@ -524,6 +524,39 @@ contract("GnosisSafe", function (accounts) {
         assert.equal(buyOrder.validFrom, currentBatch)
       }
     })
+    it("Places bracket orders and checks explicitly the expiry", async () => {
+      const masterSafe = await GnosisSafe.at(await deploySafe(gnosisSafeMasterCopy, proxyFactory, [safeOwner.account], 1))
+      const bracketAddresses = await deployFleetOfSafes(masterSafe.address, 6)
+      const baseToken = 0 // ETH
+      const quoteToken = 1 // DAI
+      const lowestLimit = 90
+      const highestLimit = 120
+      await prepareTokenRegistration(accounts[0], exchange)
+
+      await exchange.addToken(testToken.address, { from: accounts[0] })
+
+      const customExpiry = (await exchange.getCurrentBatchId.call()).toNumber() + 50
```

Comment:
Is there a reason why you don't do `exchange.getCurrentBatchId()` but you add `.call()`?

Reply:
since it is a view function, it does not make a difference.